.github/workflows/publish.yml

```diff
@@ -25,11 +25,11 @@ jobs:
           echo "Tag is: $TAG"
 
           if [[ "$TAG" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
Would we able to publish sandbox version of binary with this check? Like: v1.0.0-sandbox.
Same question arises while publishing sandbox version for extension? Like: vsix-v1.0.0-sandbox